src/modelutils.py

```diff
@@ -28,11 +28,7 @@ def skip(*args, **kwargs):
 
 def dispatch_quantized_model(model):
     num_devices = torch.cuda.device_count()
-    device_map = {
-        "model.embed_tokens": 0,
-        "model.norm": num_devices - 1,
-        "lm_head": 0 if model.config.model_type == "cohere" else num_devices - 1,
-    }
+    device_map = {"model.embed_tokens": 0, "model.norm": num_devices - 1, "lm_head": 0}
```

Comment:
Sanity check: is this intended or did you accidentally undo parts from a previous PR?

Reply:
it is intended, for some reason, I get device mismatch without doing so